### PR TITLE
Add kubevirt/kubevirt approvers to this repository's OWNERS

### PR DESCRIPTION
**Pull request by iholder101**
**What this PR does / why we need it**:
This repository lacks approvers, and most of the existing approvers aren't very active anymore.

As Kubevirt becomes more popular and active, and more design proposals are being introduced, we need more people to share the burden of reviewing and approving such proposals. This is especially important since https://github.com/kubevirt/community/pull/251 had landed.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by fabiand** [comment]
/hold

What is the justification of kubevirt/kubevirt approvers to gain approval rights in this repository?
Why not a different set of people? :)

Instead of increasing 1 core approver group with individuals, why can we not focus on the decentralized approach in order to increase our bandwidth: See #288

cc @lyarwood 

tl;dr Before simply adding more approvers, let us please fix the process to make this process more scaleable.

**Comment by iholder101** [comment]
> /hold
> 
> What is the justification of kubevirt/kubevirt approvers to gain approval rights in this repository? Why not a different set of people? :)
> 
> Instead of increasing 1 core approver group with individuals, why can we not focus on the decentralized approach in order to increase our bandwidth: See #288
> 
> cc @lyarwood
> 
> tl;dr Before simply adding more approvers, let us please fix the process to make this process more scaleable.

Hey @fabiand!
I definitely support https://github.com/kubevirt/community/pull/288. As the PR suggests, proposals (or VEPs) would be introduced to https://github.com/kubevirt/enhancements. I thought that in the meantime, until this effort takes effect, more Kubevirt approvers could contribute in reviewing "legacy" design proposals.

After we move to kubevirt/enhancements, I guess it would still makes sense that k/k approvers will have approval right in this repo, which I assume will mostly revolve around community guidelines.

**Comment by fabiand** [comment]
Let's focus on #288 and move it forward.

Merging this pr is just making us loose our focus.

**Comment by iholder101** [comment]
/remove-lifecycle stale

@fabiand, see https://github.com/kubevirt/community/pull/319.

As said above, I do support https://github.com/kubevirt/community/pull/288 and try to promote it, but I don't think it should be a blocker for this PR.
Would you consider unholding this and to get this in until your PR is merged?

**Comment by dhiller** [comment]
+1 to what @iholder101 said.

As [discussed in the community meeting](https://docs.google.com/document/d/1nE09vQWcCTW-9Ohe9oCldWrE0he-T_YFJ5D1xNzMtg4/edit#heading=h.8k5y3b9jfo4r), this repository lacks active approvers - this can be seen by the number of open pull requests without any activity from reviewer/approver side.

**Comment by iholder101** [comment]
> Let's focus on #288 and move it forward.
> 
> Merging this pr is just making us loose our focus.

@fabiand IMHO we should either split https://github.com/kubevirt/community/pull/288 into two (as suggested [here](https://github.com/kubevirt/community/pull/288#issuecomment-2338213788)) to move it forward, or get this in, or think of a different solution.

While we're not being able to converge into a solution, the problem gets bigger over time.

I think there's a pretty large consensus to get this PR in. I identify at least @dhiller @acardace @aburdenthehand @fossedihelm @alicefr @RamLavi @orelmisan @orenc1 as supporters for this approach, and we've already raised it in the community meeting and there was no pushback.

WDYT?

**Comment by dankenigsberg** [comment]
I don't believe there is consensus for mass addition of approvers, please refer to the lively discussion on https://groups.google.com/g/kubevirt-dev/c/uqN9cPRZAF8

What we need is few strong and active reviewers, such as the newly-nominated @jean-edouard . We need a plan to ensure that the existing approvers are active, and we need commitment to review and participate.

/hold

**Comment by aburdenthehand** [comment]
@iholder101 Now that #362 is merged, can we close this?

**Comment by iholder101** [comment]
> @iholder101 Now that #362 is merged, can we close this?

Yes, thank you

/close
